src/kit.ts

```diff
@@ -299,11 +300,13 @@ export async function vsInstallations(): Promise<VSInstallation[]> {
   const inst_ids = [] as string[];
   const vswhere_exe = path.join(thisExtensionPath(), 'res/vswhere.exe');
   const vswhere_args = ['-all', '-format', 'json', '-products', '*', '-legacy', '-prerelease'];
-  const vswhere_res = await proc.execute(vswhere_exe, vswhere_args).result;
+  const vswhere_res = await proc.execute(vswhere_exe, vswhere_args, null, {silent: true, encoding: 'utf8'}).result;
```

Comment:
What happened to using `cmd.exe` and `chcp`?

Reply:
Ah, I haven't added that yet, since I am still  puzzled by the VS problem.